UpSkill_App/UpSkill.Infrastructure/Data/ApplicationDbContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace UpSkill.Infrastructure.Data
+{
+    using UpSkill.Infrastructure.Data.Models;
+    using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+    using Microsoft.EntityFrameworkCore;
+
+    public class ApplicationDbContext : IdentityDbContext<ApplicationUser>
+    {
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
+            : base(options)
+        {
+        }
+
+        public DbSet<Administrator> Administrators { get; init; }
+
+        public DbSet<Owner> Owners { get; init; }
+
+        public DbSet<Employee> Employees { get; init; }
+
+        public DbSet<Coach> Coaches { get; init; }
+
```

Comment:
I think we had included Category, CourseCategory, EmployeeCourse (both Mapping Tables), LiveSession and more within the Wiki.

Reply:
I don't know how I even forgot to implement those. Thank you for mentioning it!